fast_plotter/plotting.py

```diff
@@ -140,7 +140,7 @@ def __call__(self, col, **kwargs):
                 width = self.linewidth
                 style = "-"
             else:
-                color = None
+                color = color
```

Comment:
Could you do something like `color if color else None`?

Reply:
Yeah, I can try that. Seems like an easy enough change

Reply:
Ahh, that doesn't work as intended. If there's no colour tied to a dataset, it's given one from the default colourmap (`nipy-spectral`, not `tab10`)